package.json

```diff
@@ -64,9 +64,9 @@
     "typescript": "4.1.3"
   },
   "peerDependencies": {
-    "@react-navigation/native": "5.9.2",
-    "react": "16.13.1",
-    "react-native": "0.63.4",
+    "@react-navigation/native": "^5.6.0",
+    "react": "^16.11.0",
+    "react-native": "^0.62.0",
```

Comment:
https://github.com/DataDog/dd-sdk-reactnative/issues/56#issuecomment-852895172
we had this issue with `v0.63.2`

are you making this change in response to a problem/user request?

Reply:
No, it is really just output of the test. The build issue mentioned is not a problem of DD SDK itself, but the problem of RN platform. Similar thing happens with RN `0.64.x` - in my tests I had to apply the fix https://github.com/facebook/react-native/issues/31034#issuecomment-812564390 to be able to compile.

However, after thinking a bit more, maybe it is better to revert the change for `react-native` (and maybe remove `react` from `peerDependencies` as well, since `react-native` already defines it https://github.com/facebook/react-native/blob/41ecccefcf16ac8bcf858dd955af709eb20f7e4a/package.json#L89), because `^0.0.0` syntax [means](https://docs.npmjs.com/about-semantic-versioning) that increment happens on the minor (middle) digit, so `0.70.x`, `0.80.x` will be accepted, but looking on RN history it never hit `1.x` release, while changes between minor versions can be potentially breaking.

Reply:
you could use : `>= 0.62.0 < 0.70.0` here actually

Reply:
the mentioned issue is probably caused by the fact that DD SDK (Swift pod) imports `React-Core`. it's difficult to say whose problem this is. 
using `use_frameworks!` may solve the issue (haven't tested yet), we can mention that in [`TROUBLESHOOTING.md`](https://github.com/DataDog/dd-sdk-reactnative/blob/main/TROUBLESHOOTING.md) or readme.

Reply:
btw, to build with `0.62.0` I had to add `use_frameworks!` and declare `React-Core` in `Podfile` with `modular_headers` directive like this:

```
  pod 'React-Core', :path => '../node_modules/react-native/', :modular_headers => true
```

Reply:
in the end will stick to `^0.63.4`. Although it is possible to have things working on `0.62.0`, it requires some additional changes for iOS case in the `Podfile` on the client side. And later versions should be good as well, because we are not using a lot of things from RN itself, so should be stable.